docs/en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -11,6 +11,68 @@ The examples below are using certain folder naming conventions (CSS files in `cs
 SilverStripe core modules like `cms` use a different naming convention (CSS and JavaScript files in `client/src/`).
 The `Requirements` class can work with arbitrary file paths.
 
+## Exposing static assets
+
+Before requiring a static assets file in PHP code or in a template, those assets need to be "exposed". This process allows SilverStripe projects and SilverStripe modules to make static asset files available via the web server without directly exposing PHP files.
```

Comment:
```suggestion
Before requiring static asset files in PHP code or in a template, those assets need to be "exposed". This process allows SilverStripe projects and SilverStripe modules to make static asset files available via the web server without directly exposing PHP files.
```

Worth adding something like "... from locations that would otherwise be blocked from web server access, such as the `vendor` folder"?